honeybee_vtk/to_vtk.py

```diff
@@ -91,41 +91,42 @@ def convert_sensor_grid(
             raise ValueError(
                 f'{sensor_grid.display_name} does not include mesh information. '
                 'Try again with SensorGridOptions.Sensors'
-                )
+            )
         grid_data = convert_mesh(sensor_grid.mesh)
 
     grid_data.identifier = sensor_grid.identifier
-    grid_data.display_name = sensor_grid.display_name
+    grid_data.name = sensor_grid.display_name
     grid_data.type = 'Grid'
     return grid_data
 
 
 def convert_shade(shade: Shade) -> PolyData:
-    shade_data = convert_face_3d(shade.geometry)
-    shade_data.identifier = shade.identifier
-    shade_data.display_name = shade.display_name
-    shade_data.type = 'Shade'
-    return shade_data
+    polydata = convert_face_3d(shade.geometry)
+    metadata = polydata._get_metadata(shade)
+    for key, value in metadata.items():
+        polydata.add_data(value, key, data_range=[0, 0])
```

Comment:
Should we add a method for `ad_data_from_dict`?

Reply:
I added it.